bpdm-gate/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/service/BusinessPartnerService.kt

```diff
@@ -272,4 +281,51 @@ class BusinessPartnerService(
 
     }
 
+    @Scheduled(cron = "\${cleaningService.pollingCron:-}", zone = "UTC")
+    @Transactional
+    fun checkBpnUpdates() {
```

Comment:
name of the method is a bit non-descriptive. "Check" not really conveys that it actually starts a golden record task if the check validates to true